### PR TITLE
Renaming SmartSignalsRuntimeShared

### PR DESCRIPTION
**Pull request by efvink**
New project name - SmartDetectorsApplianceShared
New namespace - Microsoft.Azure.Monitoring.SmartDetectors.MonitoringAppliance

### CODE REVIEW

File: src/deployment/SmartSignalsDeployment/mainTemplate.json [hunk 1]
```diff
@@ -173,7 +173,7 @@
             "StorageAccountName": "[variables('storageAccountName')]",
             "SendgridApiKey": "[parameters('sendgridApiKey')]",
             "AnalyticsQueryTimeoutInMinutes": "15",
-            "GlobalSmartSignalContainerUri": "https://globalsmartsignals.blob.core.windows.net/signals"
+            "GlobalSmartDetectorContainerUri": "https://globalsmartsignals.blob.core.windows.net/signals"
```
Comment (EranGon) [inline src/deployment/SmartSignalsDeployment/mainTemplate.json]: >signals [](start = 97, length = 7)

OMG

Comment (EranGon) [inline src/deployment/SmartSignalsDeployment/mainTemplate.json]: >signals [](start = 67, length = 7)

we need to create a new storage account and move all global detectors there
can you do it and update the Uri? Ask Aviram for help

Comment (EranGon) [reply src/deployment/SmartSignalsDeployment/mainTemplate.json]: or at least open a task for it

---
In reply to: [174055408](https://github.com/liadtal/Smart-Alerts/pull/95#discussion_r174055408) [](ancestors = 174055408)

Comment (efvink) [reply src/deployment/SmartSignalsDeployment/mainTemplate.json]: I opened task for it

---
In reply to: [174100129](https://github.com/liadtal/Smart-Alerts/pull/95#discussion_r174100129) [](ancestors = 174100129,174055408)

Comment (efvink) [reply src/deployment/SmartSignalsDeployment/mainTemplate.json]: I will change it after creating a new storage 

---
In reply to: [174055169](https://github.com/liadtal/Smart-Alerts/pull/95#discussion_r174055169) [](ancestors = 174055169)

**Pull request merged**